app/javascript/widget/mixins/nextAvailabilityTime.js

```diff
@@ -0,0 +1,256 @@
+import {
+  timeSlotParse,
+  defaultTimeSlot,
+} from 'dashboard/routes/dashboard/settings/inbox/helpers/businessHour.js';
+const { utcToZonedTime } = require('date-fns-tz');
```

Comment:
```suggestion
import { utcToZonedTime } from 'date-fns-tz';
```

ES6 imports style should be used